ledfx/sendspin/config.py

```
@@ -70,3 +72,32 @@ def validate_sendspin_server_url(url) -> tuple[bool, str]:
+
+
+def is_always_on(device_idx, query_devices, query_hostapis):
+    if not isinstance(device_idx, int) or device_idx < 0:
+        return False
+    try:
+        devices = query_devices()
+        hostapis = query_hostapis()
+        if device_idx >= len(devices):
+            return False
+        return hostapis[devices[device_idx]["hostapi"]]["name"] == "SENDSPIN"
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)